Slider/Assets/Scripts/Map/Ocean/NPCRotation.cs

```diff
@@ -0,0 +1,178 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NPCRotation : MonoBehaviour
+{
+    private List<string> rotationUpdates = new List<string> ();
+    [SerializeField] NPC diceGirl;
+    [SerializeField] NPC diceGuy;
+    [SerializeField] NPC fisherman;
+    [SerializeField] NPC alien;
+    [SerializeField] NPC porker;
+    [SerializeField] NPC traveling_merchant;
+    [SerializeField] NPC catBeard;
+    [SerializeField] NPC fezziwig;
+    [SerializeField] NPC amberOak;
+    [SerializeField] NPC muncher;
+    [SerializeField] Transform tile2Transform;
+    Transform tavernTransform;
+
+    public bool gotBreadge = false;
```

Comment:
~~serialize? (or initialize them in an `Awake()` like `gotBreadge = PlayerInventory.Contains("breadge", Area.Ocean)` or sm~~

lol i guess just leave a comment, in the future idt it's that bad to have more `ISavable` interfaces around the place but idrk how much itll affect performance